docs/how-tos/provision.md

```diff
@@ -0,0 +1,310 @@
+---
+title: "Configure provisioning with viam-agent"
+linkTitle: "Prepare provisioning"
```

Comment:
These names don't make sense to me, and are too easily confused with the other "provision-setup" page (if anything, the titles should be switch. THIS is the setup for provisioning, the other is using provisioning as an end-user.)

These instructions are for "pre-installation of Viam on a manufacturing (or other) image" So I'd suggest a name something like that. At least the phrase "pre-install" belongs, IMHO.